AppInspector.CLI/Writers/VerifyRulesJsonWriter.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Microsoft.ApplicationInspector.Commands;
+using Newtonsoft.Json;
+
+namespace Microsoft.ApplicationInspector.CLI.Writers
+{
+    class VerifyRulesJsonWriter : CommandResultsWriter
+    {
+        public override void WriteResults(Result result, CLICommandOptions commandOptions, bool autoClose = true)
```

Comment:
This looks like the same as in TagDiffJsonWriter, could this be the definition of WriteResults in the base class and then only override it when you have something different to do?

Reply:
It seems like you could have fewer of these CommandResultsWriter classes since so few of them differ in behavior.  Or alternately use subclasses and inheritance to only implement these behaviors once.  Its a lot of reused code.

Reply:
Yeah, that would reduce classes -see prior comments also for background but I'll do that.